doc/develop/test/ztest.rst

```diff
@@ -593,10 +593,19 @@ randomize the order. The output from the test will display the seed for failed
 tests.  For native simulator builds you can provide the seed as an argument to
 twister with `--seed`
 
-Static configuration of ZTEST_SHUFFLE contains:
 
- - :kconfig:option:`CONFIG_ZTEST_SHUFFLE_SUITE_REPEAT_COUNT` - Number of iterations the test suite will run.
- - :kconfig:option:`CONFIG_ZTEST_SHUFFLE_TEST_REPEAT_COUNT` - Number of iterations the test will run.
+Repeating Tests
+***********************
+By default the tests are executed once. The test cases and test suites
+may be executed multiple time. Enable :kconfig:option:`CONFIG_ZTEST_REPEAT` to
+multiply the execution. By default the multiplication equals three, but it can
```

Comment:
```suggestion
execute the tests multiple times. By default the multiplication factors are 3, which means every test suite is executed 3 times and every test case is executed 3 times. This can
```
For extra clarity.